packages/generators/templates/component/{{name}}.stories.{{mdx}}

```diff
@@ -0,0 +1,101 @@
+import { ArgsTable, Canvas, Meta, Story } from "@storybook/addon-docs";
+import { {{name}} } from ".";
```

Comment:
Should the `"."` be something like `"@jobber/components/{{name}};"` or `"./{{name}}`?